src/status_im/contexts/profile/settings/screens/password/change_password/loading.cljs

```diff
@@ -0,0 +1,51 @@
+(ns status-im.contexts.profile.settings.screens.password.change-password.loading
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.contexts.profile.settings.screens.password.change-password.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+;; NOTE: The duration of the password change depends on the device hardware, so to avoid
+;; quick changes in UI when mounted, we're waiting for a bit.
+(def ^:private minimum-loading-time 5000)
```

Comment:
Do we have a way of knowing or measuring the maximum time a device would need to change a password?

Reply:
It should be proportional to the size of the database so the more communities/chats/contacts/collectibles the user has, the longer it should take